.gitignore

```diff
@@ -2,4 +2,4 @@
 *.swp
 log/*
 .DS_Store
-db.sqlite3
+db.sqlite3
```

Comment:
@dannypernik Can you please comment this line again so that your PR does not include the db file? You may have to remove the file from your PR manually. Basically, this was just a one time trick to get you in sync with production and we don't want to overwrite the db in production with any local changes.